packages/templates/clients/kafka/test/__fixtures__/asyncapi-adeo.yml

```
@@ -0,0 +1,332 @@
+            Use
+            `x-value.subject.name.strategy=io.confluent.kafka.serializers.subject.RecordNameStrategy`
+            in your consumer configuration.
+    messages:
+      costingResponse:
+        $ref: "#/components/messages/costingResponse"
+
+
+
+
+operations:
+    requestCosting:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)